synfig-studio/src/gui/workarea.cpp

```diff
@@ -1405,6 +1381,20 @@ WorkArea::on_drawing_area_event(GdkEvent *event)
 				return true;
 			}
 
+			if(guide_highlighted){
+				Gtk::Menu* waypoint_menu(manage(new Gtk::Menu()));
```

Comment:
misleading variable name

Reply:
Oh yes, definitely xD. Will change to something like 'guide_menu'

Reply:
done